app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/components/ImportLoginsInstructionStep.kt

```diff
@@ -0,0 +1,153 @@
+        }
+        Spacer(Modifier.height(24.dp))
+        Text(
+            text = createClickableAnnotatedString(
+                mainString = stringResource(R.string.need_help_checkout_out_import_help),
+                highlights = listOf(
+                    ClickableTextHighlight(
+                        textToHighlight = stringResource(R.string.import_help_highlight),
+                        onTextClick = onHelpClick,
+                    ),
+                ),
+            ),
+            style = BitwardenTheme.typography.bodySmall,
+            color = BitwardenTheme.colorScheme.text.secondary,
+            modifier = Modifier.standardHorizontalMargin(),
+        )
+        Spacer(Modifier.height(24.dp))
+        BitwardenFilledButton(
+            label = stringResource(R.string.continue_text),
+            onClick = onContinueClick,
+            modifier = Modifier
+                .fillMaxWidth()
+                .standardHorizontalMargin(),
+        )
+        Spacer(Modifier.height(12.dp))
+        BitwardenOutlinedButton(
+            label = stringResource(R.string.back),
+            onClick = onBackClick,
+            modifier = Modifier
+                .fillMaxWidth()
+                .standardHorizontalMargin(),
+        )
+        Spacer(Modifier.navigationBarsPadding())
+    }
+}
+
+@Preview
+@Composable
+private fun ImportLoginsInstructionStep_preview() {
+    BitwardenTheme {
+        Column(modifier = Modifier.background(BitwardenTheme.colorScheme.background.primary)) {
+            ImportLoginsInstructionStep(
+                stepText = "Step text",
+                stepTitle = "Step title",
+                instructions = listOf(
+                    InstructionStep(
+                        stepNumber = 1,
+                        instructionText = buildAnnotatedString {
+                            append("Step text 1")
+                            withStyle(
+                                SpanStyle(
+                                    fontWeight = FontWeight.Bold,
+                                    fontFamily = BitwardenTheme.typography.bodyMedium.fontFamily,
+                                ),
+                            ) {
+                                append(" with bold text")
+                            }
+                        },
+                        additionalText = null,
+                    ),
+                    InstructionStep(
+                        stepNumber = 2,
+                        instructionText = buildAnnotatedString {
+                            append("Step text 2")
+                        },
+                        additionalText = "Added deets",
+                    ),
+                    InstructionStep(
+                        stepNumber = 3,
+                        instructionText = buildAnnotatedString {
+                            append("Step text 3")
+                        },
+                    ),
+                ).toImmutableList(),
```

Comment:
The `.immutableListO` should be on it's own line.

Or you can change `listOf` to `persistentListOf`.